README.md

```diff
@@ -1,44 +1,173 @@
 # COMP110 Worksheet 4
 
-Please edit this README.md file with your answers to the worksheet questions.
-
 ## Question 1
 
 ### a
 
+A | B | C | A AND B | NOT C | A AND B AND NOT C
+---|---|---|---|---|---
+0 | 0 | 0 |    0    |   1   |         0
+0 | 1 | 0 |    0    |   1   |         0
+0 | 0 | 1 |    0    |   0   |         0
+0 | 1 | 1 |    0    |   0   |         0
+1 | 0 | 0 |    0    |   1   |         0
+1 | 1 | 0 |    1    |   1   |         1
+1 | 0 | 1 |    0    |   0   |         0
+1 | 1 | 1 |    1    |   0   |         0
+
 ### b
 
+A | B | C | NOT C | B AND NOT C | NOT (B AND NOT C) | A AND NOT (B AND NOT C)
+---|---|---|---|---|---|---
+0 | 0 | 0 |1|0|1|0
+0 | 1 | 0 |1|1|0|0
+0 | 0 | 1 |0|0|1|0
+0 | 1 | 1 |0|0|1|0
+1 | 0 | 0 |1|0|1|1
+1 | 1 | 0 |1|1|0|0
+1 | 0 | 1 |0|0|1|1
+1 | 1 | 1 |0|0|1|1
+
 ### c
 
+A | B | C |NOT B|A OR NOT B| A OR C|(A OR NOT B) AND (A OR C)
+---|---|---|---|---|---|---
+0 | 0 | 0 |1|0|0|0
+0 | 1 | 0 |0|0|0|0
+0 | 0 | 1 |1|0|1|0
+0 | 1 | 1 |0|0|1|0
+1 | 0 | 0 |1|1|1|1
+1 | 1 | 0 |0|0|1|0
+1 | 0 | 1 |1|1|1|1
+1 | 1 | 1 |0|0|1|0
```

Comment:
It looks like your `A OR NOT B` column has actually been calculated as `A AND NOT B`, which means a few rows here are incorrect